actions/kpops-runner/action.yaml

```diff
@@ -0,0 +1,48 @@
+name: "Kpops runner"
+description: "Run Kpops command line with the given params"
+
+inputs:
+  mode:
+    description: "The command mode you want to use to launch kpops"
+    required: true
+  working-directory:
+    description: "The root directory containing the config.yaml, pipelines folder and defaults"
+    required: true
+    default: "."
+  pipeline:
+    description: "Pipeline file you want to run"
```

Comment:
if this is required (aka doesn't have a default value), could you put it first in the list of inputs